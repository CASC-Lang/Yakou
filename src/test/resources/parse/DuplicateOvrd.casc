class DuplicateOvrd

impl DuplicateOvrd {
    ovrd ovrd fn a1() {}
}