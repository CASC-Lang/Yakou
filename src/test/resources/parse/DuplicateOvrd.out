error: Duplicate `ovrd` keyword
--> ./DuplicateOvrd.casc:4:4
3 | impl DuplicateOvrd {
4 |     ovrd ovrd fn a1() {}
        ^^^^= hint: Encountered first one here
error: Duplicate `ovrd` keyword
--> ./DuplicateOvrd.casc:4:9
3 | impl DuplicateOvrd {
4 |     ovrd ovrd fn a1() {}
             ^^^^= hint: Duplicate here